front/src/components/icons/SvgIcon.jsx

```diff
@@ -0,0 +1,17 @@
+const spritePath = '/assets/icons/icon-sprite.svg';
```

Comment:
why don't you import the svg here? 

Like

import sprite from `../.../.svg` ?

Reply:
Done, but with absolute path!